packit_service/worker/build/build_helper.py

```diff
@@ -434,7 +434,15 @@ def _report(
         # We are only commenting final states to avoid multiple comments for a build
         # Ignoring all other states eg. pending, running
         if state in final_commit_states and check_names is not None:
-            self.status_reporter.report_status_by_comment(state, url, check_names)
+            self.status_reporter.report_status_by_comment(
+                state,
+                url,
+                check_names,
+                (
+                    f"{description}\n---\n"
+                    "Packit-User does not have access to project. We have requested access."
```

Comment:
What about using italics and mention the difference?
```suggestion
                    "*Packit-User does not have access to the source project. We have requested access to be able to use merge-request status instead of the comments.*"
```

Reply:
:+1: